### ISSUE TITLE
Make VBA-M not pause in menu when Linked

### ISSUE THREAD

**Issue by StrikerMan780** [visualboyadvance-m/visualboyadvance-m#83]
Currently, when trying to play online or in LAN, if someone clicks the menu, the game pauses and breaks the connection. Please make it so that when you click the menu when in a linked game, the emulation still keeps running.

**Comment by rkitover** [comment]
@StrikerMan780 I'm pretty sure 13f5afa will trivially fix this, although I don't know how to use/debug link so I haven't tested.

We will try to get a new build out soon (hopefully today.)

**Comment by StrikerMan780** [comment]
Oh, awesome. Thanks!

**Comment by rkitover** [comment]
@StrikerMan780 sorry forgot to mention that this build includes the change:

https://github.com/visualboyadvance-m/visualboyadvance-m/releases/tag/waylandplus

can you please try it and tell us if it fixes the problem?

**Comment by StrikerMan780** [comment]
No, it does not. Still pauses if you click on any of the menus.

**Comment by StrikerMan780** [comment]
You can easily test this by running two VBA-M instances, going to "Start Network Link", and starting a server on one instance, and connecting to it on the second instance with an IP of 127.0.0.1.

**Comment by rkitover** [comment]
@StrikerMan780 bummer, well thanks for explaining how to test this, I will give it a try.

**Comment by StrikerMan780** [comment]
A good game to test is Kirby and the Amazing Mirror.

**Comment by rkitover** [comment]
@StrikerMan780 just an update, what I thought would be simple to fix turns out to be actually very complicated, see "Modal Event Processing Loops" in this SO answer: http://stackoverflow.com/a/18043461

On Windows, clicking a pulldown menu starts a modal operation within Windows itself, and how I'm going to get around that in Wx I have no fucking clue yet.
